pkg/syncer/fullsync.go

```diff
@@ -463,8 +463,14 @@ func getVolumeSpecs(ctx context.Context, pvList []*v1.PersistentVolume, pvToCnsE
 
 // getVolumesToBeDeleted return list of volumeIds that need to be deleted
 // A volumeId is added to this list only if it was present in cnsDeletionMap across two cycles of full sync
-func getVolumesToBeDeleted(ctx context.Context, cnsVolumeList []cnstypes.CnsVolume, k8sPVMap map[string]string) []cnstypes.CnsVolumeId {
+func getVolumesToBeDeleted(ctx context.Context, cnsVolumeList []cnstypes.CnsVolume, k8sPVMap map[string]string, metadataSyncer *metadataSyncInformer) []cnstypes.CnsVolumeId {
 	log := logger.GetLogger(ctx)
+	// inlineVolumeMap holds the volume path information for migrated volumes which are used by Pods
+	inlineVolumeMap := make(map[string]string)
+	inlineVolumeMap, err := getInlineMigratedVolumesInfo(ctx, metadataSyncer)
+	if err != nil {
+		log.Warnf("FullSync: Failed to get inline migrated volumes. Err: %v", err)
```

Comment:
If we observe error, we need to fail here.

Reply:
Done